test/core/ExamplesUnitTests/TribitsExampleProject2_Tests.cmake

```diff
@@ -264,6 +286,130 @@ TribitsExampleProject2_find_tpl_parts(SHARED  CMAKE_PREFIX_PATH_ENV)
 ########################################################################
 
 
+function(TribitsExampleProject2_find_tpl_parts_no_optional_packages_tpls sharedOrStatic)
+
+  TribitsExampleProject2_test_setup_header()
+
+  set(tplInstallBaseDir
+    "${TribitsExampleProject2_Tpls_install_${sharedOrStatic}_DIR}")
+
+  set(testNameSuffix "")
+  set(tplLibAndIncDirsArgs "")
+  set(cmakePrefixPathCacheArg "")
+  set(cmakePrefixPathEnvArg "")
```

Comment:
This variable is not needed in this function.  Should remove.

Reply:
All of these unneeded temp vars are removed in new commit https://github.com/TriBITSPub/TriBITS/pull/474/commits/7b552d36100f5b6851cd49493eb3a5617c8c3bb1.